src/dnlib.netcore20.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <Product>dnlib</Product>
+    <RootNamespace>dnlib</RootNamespace>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AssemblyName>dnlib</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <OutputPath>..\Debug\bin\</OutputPath>
```

Comment:
Won't this overwrite the other non-netstandard2.0 file?

Reply:
No, I tested that as well. .NET Core binaries always end up in a separate "netcoreapp2.0" sub-folder under *../{Debug | Release}/bin*.